spec/features/merchants/items/index_spec.rb

```diff
@@ -101,5 +101,43 @@
         expect(current_path).to eq new_merchant_item_path(merchant_1)
       end
     end
+
+    describe 'Popular Items' do
+      it 'displays the name of the top 5 items with links to the item show page' do
+
+        item_1 = create(:item_with_successful_transaction, number_of_invoices: 5, merchant: merchant_1)
+        item_2 = create(:item_with_successful_transaction, number_of_invoices: 1, merchant: merchant_1)
+        item_3 = create(:item_with_successful_transaction, number_of_invoices: 3, merchant: merchant_1)
+        item_4 = create(:item_with_successful_transaction, number_of_invoices: 6, merchant: merchant_1)
+        item_5 = create(:item_with_successful_transaction, number_of_invoices: 2, merchant: merchant_1)
+        item_6 = create(:item_with_successful_transaction, number_of_invoices: 4, merchant: merchant_1)
+
```

Comment:
We should make sure that you're testing similarly to the shenanigans I'm testing for top customer where you create an item with a bunch of invoices and NO successful transactions to make sure it's not counting all transactions since at the moment it doesn't seem like you're testing any of the items having many unsuccessful transactions or invoices for other merchants.

Reply:
Added below, for some reason when adding that data to this test as a separate commit, git was having issues. Changes will be in the next PR.